docs/conf.py

```diff
@@ -41,16 +41,16 @@
 
 # General information about the project.
 project = u'WSGI-Kerberos'
-copyright = u'2013, Michael Komitee'
+copyright = u'2020, D. E. Shaw & Co., L.P.'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.2.0'
+version = '1.1'
```

Comment:
Might be worth changing this to use the new `__version__` attribute rather than risking it getting out of sync here. (Same goes for setup.py)